js/core/css/respec2.css

```diff
@@ -108,6 +108,10 @@ table.simple {
     margin-bottom:  0;
 }
 
+.hidden {
```

Comment:
I don't think we can add a class with a name like that.  It is far too likely to collide with something our users have.  True, their thing is likely the same, but we need to be respectful of the hosted document's namespace.  I suggesting using respec-hidden